src/components/pagination/Pagination.tsx

```diff
@@ -10,21 +10,19 @@ const Pagination: React.FC<Props> = ({ currentPage, onPageChange }) => {
   return (
     <FlexRow>
       <PageButton
-        className="btn"
         disabled={currentPage === 1}
```

Comment:
you have not fixed this comment, why you resolved it? You need to fix `disabled` statement